tests/performance_tests/performance_test_basic.py

```diff
@@ -70,8 +70,10 @@ def __str__(self) -> str:
 
         @dataclass
         class SpecifiedContract:
-            accountName: str = "c"
+            accountName: str = "eosio"
+            ownerPrivateKey: str = "EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV"
```

Comment:
I don't think this is a valid private key, try taking one from one of the other tests, or generating one locally.

Reply:
Good catch.  Addressed: [66fc7dd](https://github.com/AntelopeIO/leap/pull/702/commits/66fc7ddeecfe193f989c6a77eb758c267466eac0)